apps/web/src/features/finance/components/detail-sections/payment-basic-info.tsx

```
@@ -0,0 +1,300 @@
+									{payment.status}
+								</StatusBadge>
+							) : (
+								"Not set"
+							)}
+						</div>
+					)}
+				</div>
+				<div>
+					<label className="font-medium text-muted-foreground text-sm">
+						Platform
+					</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)